Sources/Recent.php

```diff
@@ -417,13 +417,16 @@ function RecentPosts()
 		)
 	);
 
+	// Create an array for the permissions.
+	$boards_can = boardsAllowedTo(array('post_reply_own', 'post_reply_any', 'delete_own', 'delete_any'), true, false);
```

Comment:
Can we not reuse the $permissions array?

Reply:
a good idea. Also will help with mod compatibility.